src/chatdlg.cpp

```diff
@@ -106,8 +106,26 @@ void CChatDlg::OnClearChatHistory()
 void CChatDlg::AddChatText ( QString strChatText )
 {
     // add new text in chat window
-    txvChatWindow->append ( strChatText );
-
-    // notify accessibility plugin that text has changed
-    QAccessible::updateAccessibility ( new QAccessibleValueChangeEvent ( txvChatWindow, strChatText ) );
+	
+    // analyze strChatText to check if hyperlink
+	if (strChatText.contains("https://", Qt::CaseInsensitive) | strChatText.contains("http://", Qt::CaseInsensitive)) 
+	{
+		int indx_http  = strChatText.indexOf("http", 0);
+		int indx_space = strChatText.indexOf(" ", indx_http); 
+		if (indx_space==-1) indx_space=strChatText.length();
+
+		int cutslash = (strChatText.at(indx_space-1)=='/') ? 1:0;		// drop "/" if last character of url text
+
+		QString new_strChatText = strChatText.left(indx_http)+"<a href=\""+strChatText.mid(indx_http,indx_space-indx_http-cutslash)+"\">"+strChatText.mid(indx_http,indx_space-indx_http-cutslash)+"</a> "+strChatText.right(strChatText.length()-indx_space);
+
```

Comment:
`strChatText.mid(indx_http,indx_space-indx_http-cutslash` is repeated.  It would be easier to read if it were put into a variable (`strUrl`, perhaps).

I'd also suggest, to improve readability and reduce the likelihood of dubious URLs being misread, to use a fixed width font at 110% size (or 1.1rem).

Reply:
Here is a short code modif prior I leave for a few days trip
I'll try to get connected while I'm out.
Cheers

code at:  https://we.tl/t-ZGz7CK0XqG